### DISCUSSION TITLE
Off topic: Pen support in browsers - e.g. for inpaint or sketch?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#1014] [Q&A] [answered]
Is there a way to use digital pens in automatic? E.g. utilizing a pen display like from huion or xp-pen to draw within the inpainting or sketching panels?
Asking google leads to the answer "that is not possible in browsers" yet, but maybe someone knows a way to do it?

**Comment by Tillerz** [comment]
Windows has a general "pen/tablet support", where you then can use a pen/tablet just like a mouse. Is that not working for you?

You basically move the pointer via the pen and a click on the pen button triggers a mouse click.

**Comment by cgidesign-de** [comment]
thanks for chiming in.

Use pen as mouse works fine.
Issue is "use pen as pen" so to say.
E.g. in the inpainting canvas I like to paint the mask with the pen. But it is not painting, instead it is moving the webpage.
I don't think this is specific to automatic - it seems to be a general behaviour of web browsers.
So the question is, whether there are plugins or something like that, that adds painting functionality like in a drawing app to canvas areas of a web browser.

**Reply by tkiikt** [reply]
Maybe this will help you. Try turning off the feature called Windows Ink in the tablet's settings.
[reddit](https://www.reddit.com/r/XPpen/comments/u5vrp2/why_doesnt_my_tablet_work_on_gartic_phone/)

**Comment by cgidesign-de** [comment, marked as answer]
Tried that with no luck.
In inpaint sketch I can draw with the mouse but the pen just generates erratic lines (if at all) after 10 to 20 sec. delay - it is the same for wintab and windows ink api.

Update:
MS Edge comes to the rescue.
This browser seems to have build in pen drawing functionality.
